static/js/click_handlers.js

```diff
@@ -737,6 +739,19 @@ export function initialize() {
         stream_list.toggle_filter_displayed(e);
     });
 
+    $("body").on("click", "#show-all-private-conversations-icon", (e) => {
+        e.preventDefault();
+        pm_list.show_all_private_messages_view();
+        top_left_corner.deselect_top_left_corner_items();
+        window.location.href = "#narrow/is/private";
+    });
```

Comment:
We should delete this click handler. The calls to `pm_list.show_all_private_messages_view` will end up not happening if you navigate to the view another way, like via reloading the browser window. Any logic put here actually belongs somewhere in `pm_list.js`.

Reply:
The `top_left_corner` call is done by `narrow.activate` automatically via `top_left_corner.handle_narrow_activated`, so I think should be deleted rather than moved.